bindings/node/src/index.ts

```diff
@@ -0,0 +1,76 @@
+import * as bindings from 'bindings';
```

Comment:
Just a small suggestion, but I think

```suggestion
import bindings from 'bindings';
```

would be preferable here, it’s currently only working because TS transpiles this to the same CJS `require()` call, but in proper ESM this would just be broken

Reply:
```suggestion
import bindings = require('bindings');
```

Alternatively, we can use the verbatim syntax. Then we won't need to enable 'esModuleInterop' nor emit an import helper into the compiled code.